django_rq/management/commands/rqworker.py

```diff
@@ -88,6 +104,10 @@ def handle(self, *args, **options):
                 exception_handlers=get_exception_handlers() or None,
                 default_worker_ttl=options['worker_ttl']
             )
+            if options['use_reloader']:
+                print('Reloading worker')
```

Comment:
can you not use stdout? can you use logging? this trips up people like when when doing bash scripts.